examples/bri-3/src/bri/transactions/transactions.profile.ts

```diff
@@ -0,0 +1,62 @@
+import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
+import {
+  createMap,
+  forMember,
+  ignore,
+  mapFrom,
+  Mapper,
+} from '@automapper/core';
+import { Injectable } from '@nestjs/common';
+import { Transaction } from './models/transaction';
+import { TransactionDto } from './api/dtos/response/transaction.dto';
+
+@Injectable()
+export class TransactionsProfile extends AutomapperProfile {
+  constructor(@InjectMapper() mapper: Mapper) {
+    super(mapper);
+  }
+
+  override get profile() {
+    return (mapper) => {
+      createMap(
+        mapper,
+        Transaction,
+        TransactionDto,
+
+        //remove this once the good value is available
+        forMember(
+          (d) => d.from,
+          mapFrom((s) => ''),
```

Comment:
not sure that this is needed. if @automap is not present and forMember is missing, does the mapping happen?